deploy/services-demo/create_team_members.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
is `-x` verbosity necessary?

Reply:
I like it.  If you don't let me know and I'll remove it.